packages/lodestar-beacon-state-transition/src/fast/util/attestation.ts

```diff
@@ -0,0 +1,23 @@
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {Attestation, ATTESTATION_SUBNET_COUNT} from "@chainsafe/lodestar-types";
+import {computeSlotsSinceEpochStart} from "../../util";
+import {EpochContext} from "../index";
+
+/**
+ * Compute the correct subnet for an attestation for Phase 0.
+ */
+export function computeSubnetForAttestation
```

Comment:
Can you put these as methods on `EpochContext`? (in the same vein as https://github.com/ChainSafe/lodestar/issues/1150)

`config` won't be needed as an arg as its included in `epochCtx.config`